homework4/js/main.js

```diff
@@ -0,0 +1,76 @@
+//Додати 0.1 та 0.2 так, що був правильний математичний результат
+console.log("Sum:", Math.floor((0.1 + 0.2) * 100) / 100);
+
+
+//Авторизація користувача
+
+const userLogin = prompt("Enter your login");
+const userPassword = prompt("Enter your password");
+
+if(userLogin === 'admin' && userPassword === '2pass33210'){
+    console.log("You have successfully logged in!");
+}  else if(userLogin !== 'admin' && userPassword !== '2pass33210'){
+    console.log("Such a user does not have access to the admin panel");
+} else if(userLogin !== 'admin'){
```

Comment:
Two last conditions can be moved to one
```suggestion
} else if(userLogin !== 'admin' || userPassword !== '2pass33210' ){
```